src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -561,7 +572,7 @@ private async Task InitializeAsync(bool ignoreFailures, IEnumerable<Configuratio
             }
             catch (Exception exception) when (ignoreFailures &&
                                              (exception is RequestFailedException ||
-                                             ((exception as AggregateException)?.InnerExceptions?.All(e => e is RequestFailedException) ?? false) ||
+                                             ((exception as AggregateException)?.InnerExceptions?.All(e => e is RequestFailedException || e is OperationCanceledException) ?? false) ||
```

Comment:
There is a mismatch between the `Any` logic in TryRefreshAsync and `All` logic here. We should use the same exception handling logic in all places.